content/ADR-244-extended-wearables-addressable.md

```diff
@@ -0,0 +1,59 @@
+
+## Context, Reach & Prioritization
+
+Decentraland offers a robust ecosystem supporting various NFTs, with wearables being a significant component that users can create and exchange. These wearables hold unique ownership histories, allowing users to mint and trade one or more of them. However, a problem arises when trying to differentiate between individual minted wearable items within the Decentraland metaverse accurately. The existing definition of URN, as per the standard RFC-8141, lacks the capability to distinguish between these unique minted wearables. Consequently, a URN equivalence issue arises, making it impossible to identify which specific minted item is managing. This issue stems from the fact that the URN does not include the token ID of each minted item, which is essential for identifying the user's managed item among all the minted ones.
+
+To address this limitation and enhance the representation of wearable NFT ownership, we propose an extension to the content-addressable URN format specifically tailored for minted wearables. This extension introduces an optional token ID part at the end of the URN, designed exclusively for wearables. By selectively incorporating this token ID in the URN for wearables, we enable precise identification of each minted item, ensuring that no two minted wearables share the same URN within the Decentraland ecosystem.
+
+The purpose of this update is twofold:
+
+1. **Unambiguous Wearable Identification**: The inclusion of the token ID in the content-addressable URN guarantees unambiguous identification of each minted wearable NFT item within Decentraland. With the token ID, users, content creators, and developers can confidently validate ownership of specific minted wearable assets, accurately tracking their provenance and uniqueness.
+
+2. **Enhanced Ownership Validation**: The extended URN format significantly enhances the ownership validation process for minted wearables. By leveraging the unique token ID, the Catalyst's content validation system can effectively verify and confirm ownership of individual minted wearable items, streamlining interactions within the Decentraland metaverse.
+
+Let's consider an example where a user mints two identical blue shirts from a specific V2 collection. Currently, the URNs for each NFT appear as follows:
+* **First blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a
+* **Second blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a
+
+Once this ADR takes effect, the URNs will be modified as follows:
+* **First blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a:1
```

Comment:
I thought the last part was going to be a long, globally unique value. This looks like a local id within the collection. Let's check with @hugoArregui if this is enough for on-chain validation.